syntax = "proto3";

package cakelab.arflow_grpc.v1;

option csharp_namespace = "CakeLab.ARFlow.Grpc.V1";

message ListSessionsRequest {}
